.github/ISSUE_TEMPLATE/maintenance-task.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Maintenance Task
+about: '"The code base needs cleanup, dependency updates, tests, etc...."'
+title: ''
+labels: maintenance
+assignees: ''
+
+---
+
+## What needs to happen?
+
+
+
+## What would be the consequences for not doing this?
+
+
+
+## When does this need to get done?
```

Comment:
maybe add: "why would this be an improvement"

Reply:
Simplified the 2nd question, see what you think.